### ISSUE TITLE
Python API changes should be seen by Cobbler / CLI straight away.

### ISSUE THREAD

**Issue by Timond** [cobbler/cobbler#2755]
It seems that when using the Python API to modify objects, the changes are written to JSON but are not seen by the cobbler CLI until the cobbler service is restarted.

As an example:
```
            from cobbler import api
            capi = api.CobblerAPI()
            mySystem = api.CobblerAPI().systems().find("testadd123")
            mySystem.set_status('production')
            capi.add_system(mySystem)
```

The above code works and I can successfully see the changes in the system's JSON file. If I fetch the object again via Python API it will also show the changes.

However, the CLI doesn't see the new value until I fully restart cobbler. If I don't restart cobbler, I can only see the new value via the API, just not via the CLI.

So in summary, my feature request is for changes made via Python API to be seen by cobbler (saved) straight away, and thus accessible via the Cobbler CLI.
